modules/global/src/com/haulmont/addon/ldap/service/MatchingRuleService.java

```diff
@@ -17,25 +17,32 @@
 package com.haulmont.addon.ldap.service;
 
 import com.haulmont.addon.ldap.entity.AbstractCommonMatchingRule;
+import com.haulmont.addon.ldap.entity.AbstractDbStoredMatchingRule;
+import com.haulmont.cuba.security.entity.Group;
 
 import java.util.List;
 
 public interface MatchingRuleService {
-    String NAME = "ldap_MatchingRuleService";
-
-    /**
-     * Returns the amount of existing matching rules.
-     */
-    int getMatchingRulesCount();
-
-    /**
-     * Returns the GUI representation of matching rules.
-     */
-    List<AbstractCommonMatchingRule> getMatchingRulesGui();
-
-    /**
-     * Persists the state of matching rules from LDAP Matching Rule Screen.
-     */
-    void saveMatchingRules(List<AbstractCommonMatchingRule> matchingRules, List<AbstractCommonMatchingRule> matchingRulesToDelete);
-
+	String NAME = "ldap_MatchingRuleService";
+
+	/**
+	 * Returns the amount of existing matching rules.
+	 */
+	int getMatchingRulesCount();
+
+	/**
+	 * Returns the GUI representation of matching rules.
+	 */
+	List<AbstractCommonMatchingRule> getMatchingRulesGui();
+
+	/**
+	 * Persists the state of matching rules from LDAP Matching Rule Screen.
+	 */
+	void saveMatchingRules(List<AbstractCommonMatchingRule> matchingRules, List<AbstractCommonMatchingRule> matchingRulesToDelete);
+
+	/**
```

Comment:
I think, for this particular case, we can say just

```
Returns an access group for the matching rule. The access group may be either database or predefined.
```

without implementation details